configuration/reports/reportdescriptors/dataexports/sql/dispensingData.sql

```diff
@@ -0,0 +1,139 @@
+set locationOfPrescription = LOCATION_NAME(o.value_text);
+
+update temp_dispensing_enc set dispensedLocation = location_name(location_id);
+update temp_dispensing_enc set patientIdentifier = zlemr(patient_id);
+update temp_dispensing_enc set dispensedBy = provider_name_of_type(encounter_id, @dispenser_role, 0);
+update temp_dispensing_enc set prescribedBy = provider_name_of_type(encounter_id, @ordering_provider, 0);
+update temp_dispensing_enc set dispensedLocation = encounter_location_name(encounter_id); 
+
+DROP TEMPORARY TABLE IF EXISTS temp_meds;
+CREATE TEMPORARY TABLE temp_meds
+(	encounter_id		int(11),
+	obs_id				int(11),
+	medication			varchar(255),	
+	dosage				int,
+	dosageUnits			varchar(255),
+	frequency			varchar(255),
+	duration			int,
+	durationUnits		varchar(255),
+	amount				int,
+	instructions		varchar(255));
+
+insert into temp_meds(encounter_id,obs_id)
+select o.encounter_id, obs_id 
+from obs o 
+inner join temp_dispensing_enc t on t.encounter_id = o.encounter_id 
+where o.voided = 0
+and o.concept_id = concept_from_mapping('PIH','9070');
+
+DROP TEMPORARY TABLE IF EXISTS temp_obs;
+create temporary table temp_obs 
+select o.obs_id, o.voided ,o.obs_group_id , o.encounter_id, o.person_id, o.concept_id, o.value_coded, o.value_numeric, o.value_text,o.value_datetime, o.value_coded_name_id ,o.comments 
+from obs o
+inner join temp_meds t on t.obs_id = o.obs_group_id
+where o.voided = 0;
+
+create index temp_obs_ci1 on temp_obs(obs_group_id, concept_id);
+
+update temp_meds t
+inner join temp_obs o on o.obs_group_id = t.obs_id and o.concept_id = concept_from_mapping( 'PIH','1282')
+set t.medication = concept_name(o.value_coded, @locale);
+
+update temp_meds t
+inner join temp_obs o on o.obs_group_id = t.obs_id and o.concept_id = concept_from_mapping( 'PIH','9073')
+set t.dosage = value_numeric;
+
+update temp_meds t
+inner join temp_obs o on o.obs_group_id = t.obs_id and o.concept_id = concept_from_mapping( 'PIH','9074')
+set t.dosageUnits = value_text;
+
+update temp_meds t
+inner join temp_obs o on o.obs_group_id = t.obs_id and o.concept_id = concept_from_mapping( 'PIH','9363')
+set t.frequency = concept_name(o.value_coded, @locale);
+
+update temp_meds t
+inner join temp_obs o on o.obs_group_id = t.obs_id and o.concept_id = concept_from_mapping( 'PIH','9075')
+set t.duration = value_numeric;
+
+update temp_meds t
+inner join temp_obs o on o.obs_group_id = t.obs_id and o.concept_id = concept_from_mapping( 'PIH','6412')
+set t.durationUnits = concept_name(o.value_coded, @locale);;
+
+update temp_meds t
+inner join temp_obs o on o.obs_group_id = t.obs_id and o.concept_id = concept_from_mapping( 'PIH','9071')
+set t.amount = value_numeric;
+
+update temp_meds t
+inner join temp_obs o on o.obs_group_id = t.obs_id and o.concept_id = concept_from_mapping( 'PIH','9072')
+set t.instructions = value_text;
+
+select 
+e.visitId,
+e.encounter_id,
```

Comment:
This is "encounterId" in the previous version.  Not sure if this change is intentional.